src/main/java/org/sonar/scm/git/blame/RepositoryBlameCommand.java

```diff
@@ -110,6 +112,16 @@ public RepositoryBlameCommand setFilePaths(@Nullable Set<String> filePaths) {
     return this;
   }
 
+  /**
+   * Add a callback to check the progress of the algorithm
+   * @param progressCallBack Consumer to be called each time a commit is processed by the algorithm.
+   * First parameter of the callback is the commit iteration number, second is the commit hash that is starting to be processed
```

Comment:
Maybe add a note that a commit might be processed multiple times?

Reply:
done